src/sentry/api/endpoints/project_stacktrace_link.py

```diff
@@ -258,6 +260,35 @@ def get(self, request: Request, project: Project) -> Response:
                     # When no code mapping have been matched we have not attempted a URL
                     if current_config["outcome"].get("attemptedUrl"):
                         result["attemptedUrl"] = current_config["outcome"]["attemptedUrl"]
+
+                should_get_codecov_line_coverage = (
+                    features.has(
+                        "organizations:codecov-stacktrace-integration",
+                        project.organization,
+                        actor=request.user,
+                    )
+                    and project.organization.flags.codecov_access
```

Comment:
We should prevent hitting codecov from a local setup if the CODECOV_TOKEN is not set.

```suggestion
                    and project.organization.flags.codecov_access
                    and CODECOV_TOKEN
```